docs/source/references/functional.rst

```diff
@@ -270,13 +270,21 @@ peak_signal_noise_ratio [func]
 .. autofunction:: torchmetrics.functional.peak_signal_noise_ratio
     :noindex:
 
+
 universal_image_quality_index [func]
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. autofunction:: torchmetrics.functional.universal_image_quality_index
     :noindex:
 
 
+universal_spectral_angle_mapper [func]
```

Comment:
to also keep in alphabetical order, please move above `universal_image_quality_index`
```suggestion
spectral_angle_mapper [func]
```

Reply:
I will revise this